todo/android/todo/build.gradle

```diff
@@ -23,18 +23,14 @@ android {
 
 repositories {
     mavenCentral() // For Facebook Log  in
-
-    // TODO: Remove once BSON 3.5.0 is released
-    maven {
-        url "https://oss.sonatype.org/content/repositories/snapshots"
-    }
     google()
 }
 
 dependencies {
     implementation fileTree(include: ['*.jar'], dir: 'libs')
-    implementation 'org.mongodb:stitch:3.0.0'
-    implementation 'com.facebook.android:facebook-android-sdk:4.21.1'
-    implementation 'com.android.support:design:25.0.0'
-    implementation 'com.google.android.gms:play-services-auth:11.6.0'
+    implementation 'org.mongodb:stitch-android-core:4.0.0-beta-3'
+    implementation 'org.mongodb:stitch-android-services-mongodb-remote:4.0.0-beta-3'
```

Comment:
I assume this will be updated to latest before we publish everything?

Reply:
Yep